public/js/activity_database.js

```diff
@@ -14,6 +14,7 @@ $(function() {
     { data: 'patrol_start_end_date', name: 'patrol_start_end_date' },
     { data: 'start_time', name: 'start_time' },
     { data: 'end_time', name: 'end_time' },
+    { data: 'total_time', render: function (data, type, row) {return data + ' hours'}},
```

Comment:
Ref comment about calculations, do the calculation in the back-end and not here.

Reply:
Ended up leaving it on the initial submit, so it can just be pulled as a string and does not need to be calculated every single time. When the edit patrol log update comes, it will need to be recalculated again then, and resaved as a string.

Reply:
We'll have to redo calculations in personal_profile and user_profile modules so it doesn't come to abnormalities.

I'll write up a story about that if I don't forget.